src/components/Constants/UseApiConstants.tsx

```diff
@@ -14,14 +14,15 @@ export const useApiConstants = ():
   });
 
   const userSavedLanguage = data?.user.preferences?.locale;
```

Comment:
Sorry for the wordsmithing. Testing it in preview now.

```suggestion
  const currentLanguage = data?.user.preferences?.locale;
  const [constantsLanguage, setConstantsLanguage] = useLocalStorage(
    `constants-language`,
    '',
  );
```